lib/pronto/fasterer.rb

```diff
@@ -13,6 +13,7 @@ def run
     end
 
     def inspect(patch)
+      "MAGIC MAGIC MAGIC"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.